docs/development.rst

```diff
@@ -64,31 +65,6 @@ The run info ``HASH`` stores:
   ``backend/entityservice/cache/active_runs.py`` for implementation.
 
 
-Continuous Integration Testing
```

Comment:
I removed it because we should have before. And I added `devops` in the links from this page.